Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "png"
-version = "0.16.7"
+version = "0.17.0"
 license = "MIT OR Apache-2.0"
 
 description = "PNG decoding and encoding library in pure Rust"
```

Comment:
Since `bitflags` seems to preserve the right for bumps in new versions, should we restrict our dependency to `<= 1.4.0`?

Reply:
I think that would make us incompatible with any future crate that required bitflags 1.4.0+? (because 1.3 and 1.4 are semver compatible and Cargo doesn't allow multiple versions within the same compatibility range)

Reply:
That would be much worse than the current situation where only old compilers are broken, and it can be worked around by putting "=1.2.0" in a downstream Cargo.toml

Reply:
True.. Okay, so we don't do anything but reactively advertise the MSRV. Kind of sucks but you're right, the only good way would be to influence cargo's dependency resolution, not introduce new bounds ourselves.